README.md

```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
 pod 'GRDB.swift'
 ```
 
-GRDB can be installed as a framework, or a static library.
-
-## Swift Package Manager
+### ⚠️ Important Note for CocoaPods installation
 
-The [Swift Package Manager](https://swift.org/package-manager/) automates the distribution of Swift code. To use GRDB with SPM, add a dependency to `https://github.com/groue/GRDB.swift.git`
+Due to an [issue](https://github.com/CocoaPods/CocoaPods/issues/11839) in CocoaPods, it is currently not possible to deploy new versions of GRDB to CocoaPods. The last version available on CocoaPods is 6.24.1. If you want to install later versions of GRDB using CocoaPods, please use one of the following workarounds:
```

Comment:
```suggestion
Due to an [issue](https://github.com/CocoaPods/CocoaPods/issues/11839) in CocoaPods, it is currently not possible to deploy new versions of GRDB to CocoaPods. The last version available on CocoaPods is 6.24.1. To install later versions of GRDB using CocoaPods, use one of the following workarounds:
```

I myself use and abuse "you" in documentation. But this is generally not useful.

Reply:
😄 changed it